tests/spicy/optimization/default-parser-functions.spicy

```diff
@@ -0,0 +1,24 @@
+# @TEST-EXEC: ${SPICYC} -p -o /dev/null %INPUT -D global-optimizer >log 2>&1
+# @TEST-EXEC: btest-diff log
+
+# @TEST-EXEC: ${SPICYC} -d -j -g %INPUT -D ast-codegen >ast.0 2>&1
+# @TEST-EXEC: ${SPICYC} -d -j    %INPUT -D ast-codegen >ast.1 2>&1
```

Comment:
```suggestion
# @TEST-EXEC: spicyc -p -o /dev/null %INPUT -D global-optimizer >log 2>&1
# @TEST-EXEC: btest-diff log

# @TEST-EXEC: spicyc -d -j -g %INPUT -D ast-codegen >ast.0 2>&1
# @TEST-EXEC: spicyc -d -j    %INPUT -D ast-codegen >ast.1 2>&1
```

both works, but for consistency we can use the simpler version (I know the existing tests aren't consistent either)